idb/src/main/kotlin/io/github/paulgriffith/kindling/idb/metrics/Sparkline.kt

```diff
@@ -0,0 +1,61 @@
+package io.github.paulgriffith.kindling.idb.metrics
+
+import org.jfree.chart.ChartFactory
+import org.jfree.chart.JFreeChart
+import org.jfree.chart.axis.NumberAxis
+import org.jfree.chart.ui.RectangleInsets
+import org.jfree.data.time.Millisecond
+import org.jfree.data.time.TimeSeries
+import org.jfree.data.time.TimeSeriesCollection
+import java.text.NumberFormat
+import javax.swing.UIManager
+
+fun sparkline(data: List<MetricData>, formatter: NumberFormat): JFreeChart {
+    return ChartFactory.createTimeSeriesChart(
+        /* title = */ null,
+        /* timeAxisLabel = */ null,
+        /* valueAxisLabel = */ null,
+        /* dataset = */
+        TimeSeriesCollection(
+            TimeSeries("Series").apply {
+                for ((value, timestamp) in data) {
+                    add(Millisecond(timestamp), value, false)
+                }
+            },
+        ),
+        /* legend = */ false,
+        /* tooltips = */ false,
```

Comment:
I like the tooltips since they give you a sort of xtrace functionality. I think tooltips should be turned on.